fabrics.go

```diff
@@ -26,9 +27,9 @@ type FabricVLAN struct {
 
 type ListFabricVLANsInput struct{}
 
-func (client *FabricsClient) ListFabricVLANs(*ListFabricVLANsInput) ([]*FabricVLAN, error) {
+func (client *FabricsClient) ListFabricVLANs(ctx context.Context, _ *ListFabricVLANsInput) ([]*FabricVLAN, error) {
```

Comment:
There are two reasons for this:

1) As you say, so that we can add in additional options fields without breaking users (if there are optional arguments - for example filters - it would not be necessary to populate them)

2) Uniformity of the API - it's messy to have some things take an options structure and others not, and requires consulting the documentation more often.